Bylaws.md

```diff
@@ -30,6 +30,7 @@ This Bylaw establishes the chain of club command. In the event that a captain is
 * Build Lead
 * Software Lead
 * CAD Lead
+* Outreach Coordinator
```

Comment:
@IronRiders/student-leaders 

Is this necessary? If so, would it be more appropriate to add all the appointed positions? For example, why specifically is Outreach Coordinator called out, but not Safety Captain?

Reply:
5/21 Leads Meeting: Leads okay removing this